airflow/providers/google/cloud/transfers/gcs_to_sftp.py

```diff
@@ -35,6 +35,31 @@ class GCSToSFTPOperator(BaseOperator):
     """
     Transfer files from a Google Cloud Storage bucket to SFTP server.
 
+    **Example**: ::
+
+        with models.DAG(
+            "example_gcs_to_sftp",
+            start_date=datetime(2020, 6, 19),
+            schedule_interval=None,
+        ) as dag:
+            # downloads file to /tmp/folder/subfolder/file.txt
+            copy_file_from_gcs_to_sftp = GCSToSFTPOperator(
+                task_id="file-copy-gsc-to-sftp",
+                source_bucket="test-gcs-sftp-bucket-name",
+                source_object="folder/subfolder/file.txt",
+                destination_path="/tmp/sftp",
+            )
+
+            # moves file to /tmp/file.txt
+            move_file_from_gcs_to_sftp = GCSToSFTPOperator(
+                task_id="file-move-gsc-to-sftp",
+                source_bucket=BUCKET_SRC,
+                source_object=OBJECT_SRC_2,
+                destination_path=DESTINATION_PATH_1,
```

Comment:
Can you use values here instead of undefined constants?

Reply:
I missed it. Thank you! FIxed.